src/rt/cpp/when.h

```diff
@@ -30,7 +119,7 @@ namespace verona::cpp
       assert(c.allocated_cown != nullptr);
     }
 
-    Access(cown_ptr<T>&& c) : t(c.allocated_cown), is_move(true)
+    Access(cown_ptr<T>&& c) : t(c.allocated_cown)

```

Comment:
I'm confused by this change. Why don't you need to keep it as a `is_move(true)`.

Reply:
That's a mistake. Thanks for the catch.